tests/unit/components/settings/settings.spec.js

```diff
@@ -75,4 +81,29 @@ describe('NavBar', () => {
     const btn = wrapper.find('#btn-sendRemotePeerID')
     expect(btn.exists()).toBe(true)
   })
+
+  test('Connected Edge device version is shown', () => {
+    const localEdgeVersion = '2.15.1'
+
+    const newStore = new VueX.Store({
+      modules: {
+        pnp: cloneDeep(pnpStoreModule),
+        edgeDevice: cloneDeep(edgeDevice)
+      }
+    })
+
+    newStore.state.edgeDevice.version = localEdgeVersion
+    newStore.state.pnp.peerConnectionStatus = PEER_CONNECTED
+    newStore.state.pnp.remotePeerId = '1234-1234-1234-1234-1234'
+
+    const component = mount(Settings, {
+      localVue,
+      vuetify,
+      router,
+      store: newStore
+    })
+
+    const versionElement = component.get('#version-element')
+    // expect(versionElement.find('#title').text()).toBe(localEdgeVersion)
```

Comment:
@vickywane where is the actual test assertion? This line is just commented out.

Reply:
I have improved this test suite to assert that an edge version is shown.